apps/web/src/lib/components/SideNavigation.svelte

```diff
@@ -0,0 +1,236 @@
+<script lang="ts">
+	import { AuthService } from '$lib/auth/authService';
+	import { getContext } from '@gitbutler/shared/context';
+	import { env } from '$env/dynamic/public';
+
+	const authService = getContext(AuthService);
+	const token = $derived(authService.token);
+
+	function logout() {
+		authService.clearToken();
+		window.location.href = `${env.PUBLIC_APP_HOST}cloud/logout`;
```

Comment:
For relative links we can use `goto` from `$app/navigation` i.e. `import { goto } from '$app/navigation';`

```suggestion
		goto('/cloud/logout')
```

Reply:
It's not a relative link - this one actually needs to go to a different subdomain (app.gitbutler.com).

Reply:
Ah okya, yeah for external URLs `window.location` is fine :+1: